tasks/connectors/netsparker/lib/netsparker_client.rb

```diff
@@ -23,6 +23,23 @@ def get_last_scan_vulnerabilities(schedule_id)
           JSON.parse(response)
         end
 
+        def receive_all_schedule_ids
```

Comment:
I'd change the method name to something like `get_all_schedule_ids` or `retrieve_all_scheduled_ids`.